components/style/properties/helpers.mako.rs

```diff
@@ -75,6 +75,10 @@
             }
             pub mod computed_value {
                 pub use super::single_value::computed_value as single_value;
+                % if requires_single:
+                pub use super::single_value::computed_value::T as SingleComputedValue;
```

Comment:
I don't think we need `requires_single`. Refactor the code that needs `SingleFoo` to use `single_value::computed_value::Foo`. Alternatively, unconditionally add the reexport.